privilege/privileges/privileges_test.go

```diff
@@ -1394,6 +1394,53 @@ func (s *testPrivilegeSuite) TestSecurityEnhancedModeStatusVars(c *C) {
 	}, nil, nil)
 }
 
+func (s *testPrivilegeSuite) TestRenameUser(c *C) {
+	rootSe := newSession(c, s.store, s.dbName)
+	mustExec(c, rootSe, "CREATE USER 'ru1'@'localhost'")
```

Comment:
Execute `drop user if exists ...` before creating. Otherwise, the test case may be affected by other cases in the future.

Reply:
Thanks, done.